README.md

```diff
@@ -49,6 +49,8 @@ To build the App Server and Apps, the following is required:
 
 * **NodeJS** - v14.x minimum (except v14.17.2) up to v16.x
 
+Note & TODO: Node 17+ will fail on Windows when running full zlux build, [more information in this thread.](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported). An upgrade solution needs to be applied across multiple failing components
```

Comment:
This will cause conflicts because updated V2 readme that should be V3 has some of this stuff, but it's ok. Will be simple to merge